### ISSUE TITLE
CentOS 7 ships ovirt page

### ISSUE THREAD

**Issue by jodumont** [cockpit-project/cockpit#8259]
Hi;

I did an update and seen than Machines become oVirt Machines and 4 new panels appeared.
the message : **oVirt login in progress** too

In no case I install ovirt and or cockpit-machines-ovirt

I also tried in a brand new VM CentOS and the same message appears.

Now How I could desinstall these extra package I don't and won't use ?

Or simply, how I could stop the login progress of oVirt ?

Regards!

**Comment by petervo** [comment]
If you installed with packages remove the packages cockpit-machines and cockpit-ovirt.

**Comment by jodumont** [comment]
Hi @petervo ;

thanks for you're anwser
in fact I tried inside VM with a new CentOS
and if I install `yum -y install cockpit`
I got the oVirt Machine

For my principal machine, cockpit-ovirt is not install
and I use cockit-machines for libvirtd / kvm but I gave you a chance and remove cockpit-machines than _restart cockpit services_ but the "issue" still real.

In the Cockpit WebUI I still have : oVirt login in progress

**Comment by mareklibra** [comment]
If `cockpit-ovirt` is not installed, the `cockpit-machines` is used instead.
Please be sure the `cockpit-ovirt` is not installed on the host you are referring to (see `rpm -qa|grep cockpit`).

Just `yum install cockpit` pulls neither `cockpit-machines` nor `cockpit-ovirt` automatically.

**Comment by mareklibra** [comment]
If `cockpit-ovirt` is not installed, the `cockpit-machines` is used instead.
Please be sure the `cockpit-ovirt` is not installed on the host you are referring to (see `rpm -qa|grep cockpit`).

Just `yum install cockpit` pulls neither `cockpit-machines` nor `cockpit-ovirt` automatically.

**Comment by jodumont** [comment]
list of package installed
[root@localhost ~]# yum remove cock*
Loaded plugins: etckeeper, fastestmirror
Resolving Dependencies
--> Running transaction check
---> Package cockpit.x86_64 0:155-1.el7.centos will be erased
---> Package cockpit-bridge.x86_64 0:155-1.el7.centos will be erased
---> Package cockpit-system.noarch 0:155-1.el7.centos will be erased
---> Package cockpit-ws.x86_64 0:155-1.el7.centos will be erased
--> Finished Dependency Resolution


 rpm -qa|grep cockpit
cockpit-155-1.el7.centos.x86_64
cockpit-bridge-155-1.el7.centos.x86_64
cockpit-ws-155-1.el7.centos.x86_64
cockpit-system-155-1.el7.centos.noarch

**Comment by mareklibra** [comment]
No matter there is no `cockpit-ovirt`, I guess you can still find something under `/usr/share/cockpit/ovirt`, same as under `... /cockpit/apps`.

If so, this is a known bug found yesterday for cockpit-155 on centos. It shall be fixed already, @martinpitt any expectation about timing, please?

**Comment by jodumont** [comment]
in fact `/usr/share/cockpit/ovirt` exist
but no ovirt under `/usr/share/cockpit/apps/`

than when to fix will be push, I'll just have to do a yum update cockpit ?

**Comment by martinpitt** [comment]
Right, I pushed the fix to the rhel dist-git yesterday. It's quite simple:
```diff
--- a/cockpit.spec
+++ b/cockpit.spec
@@ -235,8 +235,8 @@ rm %{buildroot}/%{_libexecdir}/cockpit-stub
 touch kubernetes.list
 %endif
 
-# On RHEL, apps/ovirt are not currently built
-%if 0%{?rhel}
+# On RHEL and CentOS, apps/ovirt are not currently built
+%if 0%{?rhel}{?centos}
 rm -rf %{buildroot}/%{_datadir}/%{name}/apps %{buildroot}/%{_datadir}/%{name}/ovirt
 %endif
```

@jodumont,  as a workaround, do `sudo rm -r /usr/share/cockpit/ovirt /usr/share/cockpit/apps/` for now.

I'm not sure if one can do uploads to CentOS directly. If not, the fix will trickle in with RHEL 7.4.5 around February (7.4.4 is around the corner, but already frozen, so I can't squeeze it in there any more).

**Comment by jodumont** [comment]
woot woot ;)

so again on my vm
`yum -y install cockpit`
oVirt is present
then `rm -rf /usr/share/cockpit/ovirt /usr/share/cockpit/apps/`
make it disapear


install KVM + cockpit-machines
`yum -y install qemu-kvm libvirt virt-install bridge-utils cockpit-machines cockpit-bridge`

then I found my old friend **Virtual Machines** instead of **oVirt**.

thanks a lot @mareklibra & @martinpitt for this follow and solution.
